oap-server/server-storage-plugin/storage-jdbc-hikaricp-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/jdbc/SQLExecutor.java

```diff
@@ -51,6 +55,10 @@ public void invoke(Connection connection) throws SQLException {
             LOGGER.debug("execute sql in batch: {}, parameters: {}", sql, param);
         }
         preparedStatement.execute();
+
+        if (additionalSQLs.size() > 0) {
+            invokeAdditionalSQL(connection);
+        }
```

Comment:
Recursively invoke the additional sql might avoid problem like https://github.com/apache/skywalking/pull/9098/files#r877084495

```suggestion
        for (SQLExecutor sqlExecutor : additionalSQLs) {
            sqlExecutor.invoke(connection);
        }
```